package.json

```diff
@@ -424,6 +425,7 @@
   "workspaces": {
     "packages": [
       "packages/*",
+      "packages/!(grafana-icons)/**",
```

Comment:
This will exclude the icons package from being versioned and published by lerna.

Reply:
Looks like this requires upgrading to lerna v8 to work.